polling_stations/apps/file_uploads/views.py

```diff
@@ -71,10 +89,24 @@ def get(self, request, *args, **kwargs):
 
     def get_context_data(self, **context):
         context["council"] = (
-            Council.objects.all()
+            Council.objects.using(LOGGER_DB_NAME)
+            .all()
             .exclude(council_id__startswith="N09")
             .get(council_id=self.kwargs["gss"])
         )
+        upcoming_election_dates = EveryElectionWrapper(
+            council_id=self.kwargs["gss"]
+        ).get_future_election_dates()
```

Comment:
If a council has an upcoming election that it manages, but isn't for that council then I'm not sure it will get to the list. E.g. `https://elections.democracyclub.org.uk/elections/parl.southend-west.by.2022-02-03/` doesn't appear in `https://elections.democracyclub.org.uk/api/elections.json?organisation_identifier=SOS&future=1` 

Which is where the upcoming dates come from in: https://github.com/DemocracyClub/UK-Polling-Stations/blob/993e1084698d5ff0f8079a815df16bcd71b701c1/polling_stations/apps/data_finder/helpers/every_election.py#L75 populating `self.elections` with: https://github.com/DemocracyClub/UK-Polling-Stations/blob/993e1084698d5ff0f8079a815df16bcd71b701c1/polling_stations/apps/data_finder/helpers/every_election.py#L44

I'm not yet sure how to fix this as I'm not sure we store a map of local authority -> managed elections anywhere :thinking:

Reply:
https://github.com/DemocracyClub/EveryElection/pull/1336 is an attempt to fix this. If that's a reasonable approach then the change to `get_future_election_dates()` shouldn't be too hard.